modules/installation-network-user-infra.adoc

```diff
@@ -183,6 +183,11 @@ The {op-system-base} KVM host must be configured to use bridged networking in li
 ====
 endif::ibm-z-kvm[]
 
+[IMPORTANT]
+====
+When IPsec is enabled on your {product-title} cluster, you must add the UDP 500 IPSEc IKE packets with the UDP 4500 IPsec NAT-T packets on all of your nodes.
```

Comment:
I not entirely sure about this blurb, we might as well add the ports to the regular list below.

Also `IPsec` is always spelled with lowercase `sec`